modules/articles/client/config/articles.client.menus.js

```diff
@@ -1,8 +1,17 @@
-'use strict';
+/**
```

Comment:
Comments are good, but this is unnecessary. If at any time you want to know what the file is, you can just look at the title, but the comment only helps if you can see it and anyway it doesn't add anything to the code.